src/FsAutoComplete.Core/CommandResponse.fs

```diff
@@ -248,6 +264,11 @@ module CommandResponse =
 
   let info (serialize : Serializer) (s: string) = serialize { Kind = "info"; Data = s }
   let error (serialize : Serializer) (s: string) = serialize { Kind = "error"; Data = s }
```

Comment:
This should be updated to set `ResponseError<unit>` as a Data, instead of the string